indico/modules/events/abstracts/util.py

```diff
@@ -371,3 +372,27 @@ def get_roles_for_event(event):
     roles['*']['reviewer'] = [reviewer.id for reviewer in event.global_abstract_reviewers]
     roles['*']['convener'] = [reviewer.id for reviewer in event.global_conveners]
     return roles
+
+
+def make_review_form(event):
```

Comment:
So far we always had these functions in util.py but forms.py wouldn't be a bad place either